packages/fetch/index.test.js

```diff
@@ -9,7 +9,7 @@ import {
   createPassThroughStream
 } from '@datastream/core'
 
-import { fetchResponseStream, fetchSetDefaults } from '@datastream/fetch'
+import { fetchResponseStream, fetchSetDefaults } from './index.js'
```

Comment:
Using `@datastream/fetch` is by design to allow testing using different exported variations